administrator/language/en-GB/plg_system_webauthn.ini

```diff
@@ -35,7 +35,7 @@ PLG_SYSTEM_WEBAUTHN_HEADER="Passkey Login"
 PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR="Generic Passkey"
 PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR_LABEL="%s added on %s"
 PLG_SYSTEM_WEBAUTHN_LOGIN_DESC="Login without a password using the browser passkey standard in compatible browsers. You need to have already set up passkey authentication in your user profile."
-PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Passkey"
+PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Sign in with a passkey"
```

Comment:
not a fan of this change - its overly verbose and I assume you only did it to make the word passkey lowercase which is not required in this case

Reply:
Its requires by the style guide, the reason is awareness and I think thats the Main reason for the change to passkeys

Reply:
please posint to the part of the styleguide that requires this - I didnt see it

Reply:
sorry - i saw it now